pyr2.html

```diff
@@ -58,31 +56,38 @@
 var a13 = qnbank[Qline][7];
 var snod = qnbank[Qline][8];
 var a14 = qnbank[Qline][9];
-document.write("<DL>");
+
+document.innerHTML="<DL>";
```

Comment:
`document.innerHTML` is not an actual property... you might be trying to set `document.body.innerHTML` but that would be ill-advised as it overwrites anything else in the `<body>` element.